src/garage/tf/algos/npo.py

```diff
@@ -562,7 +594,7 @@ def _build_entropy_term(self, i):
 
         return policy_entropy
 
-    def _fit_baseline(self, samples_data):
+    def _fit_baseline_after(self, samples_data):
```

Comment:
please find a way to structure this so that the baseline fitting routine is shared between the cases.